### ISSUE TITLE
MICROSOFT Chat cards cannot be accessed using enter key in NVDA

### ISSUE THREAD

**Issue by KattaBhargav** [microsoft/BotFramework-WebChat#4311]
### PWD Impact:
Can cause significant disruption. Major issue for keyboard and screen reader users. When a user enters the chat pane they cannot ever interact with interactive chat message content such as links or buttons pictures above. 
For screen reader users, they receive additional audio information which describes the controls including to press enter to interact with these messages. Sighted keyboard users do not receive any equivalent instruction. Beyond this, when pressing enter when instructed, this does nothing so keyboard users cannot interact with any chat messages such as the one pictured above. They can neither be read out or navigated. This issue only affects NVDA users and occurs across multiple browsers. Because of this, we believe this to be a compatibility issue for NVDA rather than a problem with the product. Controls and experience are as expected for JAWS users.

### Repro Steps:
1. Turn on NVDA and open URL https://leedssis.flexsocial.co.uk/smartchat.
2. Tab from the chat textbox to the hints card.
3. When you hear the words "press enter to interact"
4. Press enter.

### Actual Result:
Chat cards cannot be accessed using enter key in NVDA

### Expected Result:
The focus should move to the first hint button.

### MAS Reference:
[MAS 2.1.1 – Keyboard](https://aka.ms/MAS2.1.1)

### Test Environment:
OS: Windows 11 Version 21H2 Build 22000.675
URL: https://leedssis.flexsocial.co.uk/smartchat.
Tools: NVDA Version 2021.3.5

**Comment by KattaBhargav** [comment]
#A11yMAS;#A11ySev2;#HCL;#Accessibility;#BotFrameworkWebChat-Aug21;#Linked:Bug193992;#LinkedBug;

**Comment by KattaBhargav** [comment]
Bug copied from Azurebot team Bug id:-[BUG 193992](https://dev.azure.com/CSS-IRT/CSS%20IRT/_workitems/edit/193992)

**Comment by compulim** [comment]
I believe NVDA scan mode is interfering with navigation. Press <kbd>CAPS</kbd> + <kbd>SPACEBAR</kbd> will toggle scan mode. When scan mode is off, <kbd>ENTER</kbd> key is working.

**Comment by compulim** [comment]
Closing as external. As Prateek pointed out, this is an issue on NVDA and Narrator but not JAWS.

NVDA and Narrator did not turn off scan mode (a.k.a. browse mode) when inside the chat history.
